pkg/autoscaler/autoscaler.go

```diff
@@ -218,6 +218,8 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (desiredPodCount
 			logger.Infof("Increasing pods from %d to %d.", originalReadyPodsCount, desiredPanicPodCount)
 			a.panicTime = &now
 			a.maxPanicPods = desiredPanicPodCount
+		} else if desiredPanicPodCount < a.maxPanicPods {
+			logger.Infof("Skipping decrease from %d to %d.", a.maxPanicPods, desiredPanicPodCount)
```

Comment:
`Debugf`, perhaps? It'll still log in tests

Reply:
Done